RockLib.Messaging.CloudEvents/Directory.Build.props

```diff
@@ -5,15 +5,15 @@
 	<PropertyGroup>
 		<AnalysisMode>all</AnalysisMode>
 		<Authors>Rocket Mortgage</Authors>
-		<Copyright>Copyright 2024 (c) Rocket Mortgage. All rights reserved.</Copyright>
+		<Copyright>Copyright 2025 (c) Rocket Mortgage. All rights reserved.</Copyright>
 		<LangVersion>latest</LangVersion>
 		<Nullable>enable</Nullable>
-		<TargetFrameworks>net6.0;net48;net8.0</TargetFrameworks>
```

Comment:
Should we do these updates in the same pr we bump the version for the RockLib.Messaging base?

Reply:
Yes, because otherwise we'll get compilation errors.